polkadot/runtime/westend/src/lib.rs

```diff
@@ -191,15 +192,30 @@ parameter_types! {
 
 #[derive_impl(frame_system::config_preludes::RelayChainDefaultConfig)]
 impl frame_system::Config for Runtime {
+	type RuntimeOrigin = RuntimeOrigin;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type Hashing = BlakeTwo256;
+	type Lookup = AccountIdLookup<AccountId, ()>;
+	type RuntimeTask = RuntimeTask;
+	type PalletInfo = PalletInfo;
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type OnSetCode = ();
+	type SingleBlockMigrations = ();
+	type MultiBlockMigrator = ();
+	type PreInherents = ();
+	type PostInherents = ();
+	type PostTransactions = ();
```

Comment:
```suggestion
```

Reply:
<img width="1184" alt="Screenshot 2024-05-03 at 19 20 49" src="https://github.com/paritytech/polkadot-sdk/assets/56865496/b2f9d6bf-cf6c-459e-b1c7-c30b9f93995c">

The issue with removing these changes is that the runtime demands that they are implemented. These types should not be needed but I am not sure why the compiler shows this error. Are there any recommendations from your end @bkchr

Reply:
These have been removed, the errors were from rust-analyzer and project still builds